README.md

```diff
@@ -57,18 +61,7 @@ The config plugin has the following options:
   </tr>
 </table>
 
----
-
-### Bare React Native app
-
-To use this library, you need to have [Expo Modules installed](https://docs.expo.dev/bare/installing-expo-modules/):
-
-```
-npx install-expo-modules
-npx expo install expo-music-picker
-```
-
-#### Configure for iOS 🍏
+## Configuration for iOS 🍏
```

Comment:
I'd add these notes as plugin configuration heading has the same level as platform-specific headings.
Inspired by [expo-image-picker README](https://github.com/expo/expo/tree/main/packages/expo-image-picker#configure-for-ios)

```suggestion
## Configuration for iOS 🍏

> This is only required for usage in bare React Native apps.
```